reconcile/resource_scraper.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import logging
+from typing import cast
+
+from reconcile import queries
+
+from reconcile.status import ExitCodes
+from reconcile.utils.oc import OC_Map
+from reconcile.utils.vault import _VaultClient, VaultClient
+
+
+QONTRACT_INTEGRATION = "resource-scraper"
+
+
+def run(dry_run, namespace_name, resource_kind, vault_output_path):
```

Comment:
Thanks for adding this Maor! Can you also add support for filtering on labels? The way we were scraping the secrets was to filter on the label:
```
oc get secret ... -l app=openshift-sre-token-scraper
```

Reply:
contributions are welcome? :)

Reply:
After discussing with Dustin we are going to merge as-is, and Dustin will contribute a patch to add filtering on a later MR.